web_src/less/_base.less

```diff
@@ -1300,3 +1287,24 @@ i.icon.centerlock {
 .ui.sub.header {
     text-transform: none;
 }
+
+.ui.tabular.menu {
+    min-height: 36px;
+
+    .item {
+        padding: 0 .75rem;
```

Comment:
```suggestion
        padding-right: 12px;
        padding-left: 12px;
```

Reply:
The original top/bottom padding is a bit too much, needs reduction:

<img width="191" alt="image" src="https://user-images.githubusercontent.com/115237/85234463-d89d8d80-b40d-11ea-95b9-ae32b488524a.png">

Reply:
It looks visually big on arc-green but is fine on default styling

Reply:
Let's keep Fomantic changes to minimum however please?

Reply:
How about
```suggestion
        padding: 12px;
```